dd-sdk-android/src/main/kotlin/com/datadog/android/privacy/Consent.kt

```diff
@@ -0,0 +1,34 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.privacy
+
+/**
+ * The Consent enum class providing the possible values for the Data Privacy Consent flag.
+ * @see Consent.GRANTED
+ * @see Consent.NOT_GRANTED
+ * @see Consent.PENDING
+ */
+enum class Consent {
+    /**
+     * The permission to persist and dispatch data to the Datadog Endpoints was granted.
+     * Any previously stored pending data will be marked as ready for sent.
+     */
+    GRANTED,
+
+    /**
+     * Any previously stored pending data will be deleted and. Any Log, Rum, Trace event will
+     * be dropped from now on without persisting it in any way.
```

Comment:
There are some words missing in the first sentence or the `.` is mistaken.

Reply:
Oups, sorry ;)